ds-prep-python-loops/main.py

```diff
@@ -0,0 +1,33 @@
+record = (1, "Grimdiana", "Bones", "boulders")
+row = ""
+for x in record:
+  print(x)
+for x in record:
+  for y in row:
+    print(x , "," , y)
```

Comment:
For these steps, please build up a single string in the variable `row` through concatenation, then `print` the value in `row` once at the end:

---

![image](https://user-images.githubusercontent.com/16143771/149633220-f0d48375-9049-4da0-ac69-1bbedf888c0d.png)